CUETools/CUETools.TestCodecs/FlakeWriterTest.cs

```diff
@@ -79,24 +79,24 @@ public void ConstructorTest()
             AudioBuffer buff = Codecs.WAV.AudioDecoder.ReadAllSamples(new Codecs.WAV.DecoderSettings(), "test.wav");
             AudioEncoder target;
 
-            target = new AudioEncoder(new EncoderSettings() { PCM = buff.PCM, EncoderMode = "7" }, "flakewriter0.flac");
+            target = new AudioEncoder(new EncoderSettings() { PCM = buff.PCM, EncoderMode = "7" }, "Flakewriter0.flac");

```

Comment:
No need to change the file name of `flakewriter0.flac` in the `FlakeWriterTest`. It should stay lower case, in order to avoid unnecessary changes, which are not related to the issue.